src/main/scala/nl.knaw.dans.easy.archivebag/EasyArchiveBag.scala

```diff
@@ -81,19 +82,26 @@ object EasyArchiveBag extends Bagit5FacadeComponent with DebugEnhancedLogging {
   }
 
 
-  private def getBagSequence(bagId: BagId)(implicit ps: Parameters): Try[Option[String]] = {
-    Try {
-      val http = createHttpClient(ps.bagIndexService.getHost, ps.bagIndexService.getPort, "", "")
-      val get = new HttpGet(ps.bagIndexService.resolve(s"bag-sequence?contains=$bagId"))
-      get.addHeader("Accept", "text/plain;charset=utf-8")
-      val sw = new StringWriter()
-      val response = http.execute(get)
-      if (response.getStatusLine.getStatusCode != HttpStatus.SC_OK) throw new IllegalStateException(s"Error retrieving bag-sequence for bag: $bagId")
-      IOUtils.copy(response.getEntity.getContent, sw, "UTF-8")
-      Some(sw.toString)
+  private def getBagSequence(bagId: BagId)(implicit ps: Parameters): Try[Option[String]] = Try {
```

Comment:
Why do you need the Option here? I don't see where you return a possible None...